lms/templates/admin/instance.html.jinja2

```diff
@@ -46,6 +46,13 @@
 <form method="POST" action="{{ request.route_url("admin.instance.id", id_=instance.id) }}">
     <input type="hidden" name="csrf_token" value="{{ get_csrf_token() }}">
 
+    <fieldset class="box">
+        {{ macros.settings_textarea(instance, "Notes", "hypothesis", "notes") }}
```

Comment:
This was embarrassingly removed here https://github.com/hypothesis/lms/pull/4726/files#diff-fd5bcca6d2ee9b73ec82594547d82c2d0682f22cc2fedbb9f8ee5aac9e355015